prog/types.go

```diff
@@ -640,6 +668,17 @@ func (t *UnionType) isDefaultArg(arg Arg) bool {
 	return a.Index == 0 && isDefault(a.Option)
 }
 
+func (t *UnionType) Alignment() uint64 {
```

Comment:
If we have TypeCommon.TypeAlign, I think we don't need this. TypeCommon.Alignment should return the correct value for UnionType as well.

Reply:
Fixed.